app/views/chargeback/_cb_rate_edit.html.haml

```diff
@@ -36,4 +36,10 @@
     = _('Rate Details')
   %strong
     = _('* Caution: The value Range end will not be included in the tier.')
+    %br
+    = _('* Caution: Allocated CPU Count, Used CPU, Used Disk I/O and Used Network I/O are not supported for Chargeback for Images.')
+    %br
+    = _('* Caution: Allocated CPU Count/Cores, Used CPU, Used Disk I/O and Allocated Memory are not supported for Chargeback for Projects. ')
+    %br
+    = _('* Caution: Allocated/Used CPU Cores are not supported for Chargeback for Vms. ')
```

Comment:
Could we just rephrase these into a single sentence on a single line?

Reply:
I don't understand how. These are three different sentences how would you like to put it into one ?

Reply:
keyword: rephrase :wink:

Reply:
There are three things: Chargeback for Images, Projects, Vms. We need to inform which fields are not supported for each of them. How would you put so much info into the one line to keep readability? Anyway, I see later comments and it looks like there will be completely different solution to what I was told to do here.

Reply:
Well, for me it looks weird to have a lot of warning signs...something like this in real life:
![](http://www.signs4safety.com/SIGNS4SAFETYCATALOG/New%20Folder/automotivesigns5.jpg)

Reply:
Well, four lines of "Caution" is also too much for me but here it is not possible to put all the info in one line and to keep readability, it's too much info to put it just into one line. Anyway, I was told to do so.. Aaand.. now I was told to do it completely differently so don't worry about this, I will udpate the PR soon! ;)